eplus_rmd/translator.py

```diff
@@ -22,10 +43,787 @@ def validate_input_contents(input_json: Dict):
+                        tilt_column = cols.index('Tilt [deg]')
+                        u_factor_with_film_column = cols.index('U-Factor with Film [W/m2-K]')
+                        for surface_name in surface_names:
+                            construction_name = rows[surface_name][construction_name_column]
+                            gross_area = float(rows[surface_name][gross_area_column])
+                            azimuth = float(rows[surface_name][azimuth_column])
+                            tilt = float(rows[surface_name][tilt_column])
+                            u_factor_with_film_string = rows[surface_name][u_factor_with_film_column]
+                            u_factor_with_film = 0
+                            if u_factor_with_film_string:
+                                u_factor_with_film = float(u_factor_with_film_string)
+                            if tilt > 120:
+                                surface_classification = 'FLOOR'
+                            elif tilt >= 60:
+                                surface_classification = 'WALL'
+                            else:
+                                surface_classification = 'CEILING'
+                            if is_exterior:
+                                adjacent_to = 'EXTERIOR'
+                            else:
+                                adjacent_to = 'INTERIOR'
+                            surface = {
+                                'id': surface_name,
+                                'classification': surface_classification,
+                                'area': gross_area,
+                                'tilt': tilt,
+                                'azimuth': azimuth,
+                                'adjacent_to': adjacent_to,
+                                'does_cast_shade': do_surfaces_cast_shadows
+                            }
+                            if not is_exterior:
+                                adjacent_surface = self.get_adjacent_surface_for_each_surface()
+                                if surface_name in adjacent_surface:
+                                    adjacent_surface = adjacent_surface[surface_name]
+                                    if adjacent_surface in self.surfaces_by_zone:
+                                        surface['adjacent_zone'] = self.surfaces_by_zone[adjacent_surface]
+                            if surface_name in subsurface_by_surface:
+                                surface['subsurfaces'] = subsurface_by_surface[surface_name]
+                            surfaces[surface_name] = surface
+                            if construction_name in constructions:
+                                surface['construction'] = constructions[construction_name]
+                                if u_factor_with_film_string:
+                                    surface['construction']['u_factor'] = u_factor_with_film
+        # print(surfaces)
+        return surfaces
+
+    def gather_infiltration(self):
```

Comment:
There might be some similarities between some of these functions, but they are broken into logical pieces and everything looks good as-is.